torchbenchmark/canary_models/llama_v2_13b/__init__.py

```diff
@@ -0,0 +1,15 @@
+from torchbenchmark.tasks import NLP
```

Comment:
just curious, these models are in `canary_models` as opposed to `models` dir. Does this effect how these models are pulled in either from torchbench or dyanmorunner and if so in what way?

Reply:
This error message explains it super well, should be no difference

```
  File "/var/lib/jenkins/workspace/benchmarks/dynamo/torchbench.py", line 302, in load_model
    raise ImportError(f"could not import any of {candidates}")
ImportError: could not import any of ['torchbenchmark.models.stable_diffusion', 'torchbenchmark.canary_models.stable_diffusion', 'torchbenchmark.models.fb.stable_diffusion']
ERROR
```